reference/glossary.md

```diff
@@ -4,13 +4,17 @@ This glossary defines the terminology used to describe the various concepts rela
 
 **Actuator**: An *actuator* is a *node* representing a real robotics actuator such as a motor, a LED, a radio emitter, etc. *Actuators* may receive commands from a *controller*. Their behavior will affect the outcome of a simulation. Some *actuators* are also *sensors* if they make sensory measurement, such as a motor than can provide position or force feedback.
 
+**Ancestor node**: An *ancestor node* is a *node* that hierarchically includes another *node* in the *scene tree*.
+

```

Comment:
It sound wrong to use ancestor and descendant without the word node. We never do it in the documentation.

If you want to define the ancestor node based on the descendant node definition is ok for me.
But this definition doesn't seems very consistent with descendant node description.
Then I would prefer something like:
*The converse notion of a descendant node.*